server/controllers/Users.js

```diff
@@ -20,30 +21,44 @@ usersController.post('/registration', async (req, res) => {
     await UserService.registerUser(userData)
     res.json(userData)
 })
+usersController.post('/check-email/:email', async (req, res) => {
```

Comment:
```suggestion
usersController.post('/email/:email', async (req, res) => {
```
Implicitly, it's enough to write the path like this. You can refer to this article https://restfulapi.net/resource-naming/